src/SecTester.Bus/Dispatchers/RateLimitedHandler.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.RateLimiting;
+using System.Threading.Tasks;
+
+namespace SecTester.Bus.Dispatchers;
+
+[ExcludeFromCodeCoverage]
+internal sealed class RateLimitedHandler
+  : DelegatingHandler, IAsyncDisposable
+{
+  private readonly RateLimiter _rateLimiter;
+
+  public RateLimitedHandler(RateLimiter limiter)
+    : base(new HttpClientHandler()) => _rateLimiter = limiter;
```

Comment:
```suggestion
       : base(new HttpClientHandler()
      {
        AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
      } 
    ) => _rateLimiter = limiter;
```
Please enable decompression at least for GZip